test/test.h

```diff
@@ -84,12 +84,14 @@ using LiveTrafficCustomize = std::function<void(valhalla::baldr::GraphReader&,
 void customize_live_traffic_data(const boost::property_tree::ptree& config,
                                  const LiveTrafficCustomize& setter_cb);
 
+#ifdef DATA_TOOLS
```

Comment:
these two functions rely on GraphTileBuilder from mjolnir which is disabled when data_tools are disabled